samples/show-scale-bar/README.md

```diff
@@ -0,0 +1,37 @@
+# Show scale bar
+
+Add a scale bar to visually gauge distances on a map.
+
+![Image of show scale bar](show-scale-bar.png)
+
+## Use case
+
+Allows a user to have a visual reference for distances when navigating the map.
+
+## How to use the sample
+
+Zoom in or out of the map. The scale bar will automatically display the appropriate scale based on zoom level. Units can be in metric and/or imperial based on locale/optional settings.
+
+## How it works
+
+1. Create an `ArcGISMap` and add it to a `MapView` composable.
+2. Listen and track callback changes from `onUnitsPerDipChanged`, `onViewpointChangedForCenterAndScale`, `onSpatialReferenceChanged` using the composable `MapView`.
+3. Add the `Scalebar` composable positioned on top of the `MapView`
+4. Pass in the latest values of `unitsPerDip`, `viewpoint`, `spatialReference` & use a preferred `maxWidth` into the `ScaleBar`.
```

Comment:
```suggestion
4. Pass in the latest values of `unitsPerDip`, `viewpoint`, `spatialReference` and use a preferred `maxWidth` into the `ScaleBar`.
```

Maybe best to keep the README syntax simple in case there's any MD rendering issues on device